### ISSUE TITLE
Develop Agancy feature

### ISSUE THREAD

**Issue by natanzi** [WoollyinWalesIT/jomres#193]
As you know all hotels in the world collaboration with tourism Agency, and get discount to Agency for provides travel and tourism related services to the hotels. so we need define specials Tariff for them (discount)... but for some of them, hotel ask to pay full price or deposit base on account in sense that when its come to hotel website agency can be chose rooms and : 
1-pay total price
2- Make a payment as a deposit (The amount of money determined by the hotel)
3-payment base on balance on her jomres account
In summary I mean its grate if every user have balance in her account and its very useful for developing other Value Added Service (VAS) for jomres ...
Much Appreciate
BR
Milad
